### ISSUE TITLE
Search-results enumeration arrows should have an accessible label

### ISSUE THREAD

**Issue by sigurdm** [dart-lang/pub-dev#7246]
![image](https://github.com/dart-lang/pub-dev/assets/8613953/c7abe1a7-4790-4904-894e-1d73fea0f31a)
